project.clj

```diff
@@ -53,22 +51,18 @@
                  ;; send their logs to logstash, so we include it in the jar.
                  [net.logstash.logback/logstash-logback-encoder]
 
-                 [puppetlabs/jruby-utils "2.1.0"]
-                 [puppetlabs/jruby-deps "9.2.0.0-1"]
-
-                 ;; Update to flatland usable in jdk11
-                 [org.flatland/useful "0.11.6"]
-                 [org.flatland/ordered "1.5.7"]
```

Comment:
Did we update the thing that was pulling in old versions of these?

Reply:
yeah, older flatlands were pulled in by circleci/clj-yaml